src/main/java/org/cactoos/collection/Sticky.java

```diff
@@ -52,23 +51,16 @@ public Sticky(final E... items) {
      * @param items The array
      */
     public Sticky(final Iterable<E> items) {
-        this(new CollectionOf<>(items));
-    }
-
-    /**
-     * Ctor.
-     * @param list The iterable
-     */
-    public Sticky(final Collection<E> list) {
         super(
-            new org.cactoos.scalar.Sticky<>(
-                () -> {
-                    final Collection<E> temp = new ArrayList<>(list.size());
-                    temp.addAll(list);
-                    return Collections.unmodifiableCollection(temp);
-                }
+            new CollectionOf<>(
+                new org.cactoos.scalar.Sticky<>(
+                    () -> {
+                        final Collection<E> temp = new LinkedList<>();
+                        items.forEach(temp::add);
```

Comment:
@victornoel please, change the parameter's name `items` to `src` to be more homegeneus.